DESIGN-PROPOSALS.md

```diff
@@ -0,0 +1,93 @@
+# Design Proposal Template
+
+**Authors**: alice alice@example.com, bob jones [bob@example.com](mailto:bob@example.com)
+
+**Begin Design Discussion**: YYYY-MM-DD
+
+**(optional) Status: **implementable, accepted, deferred, rejected, withdrawn, or replaced
+
+**Checklist**:
+
+- [ ] Prerequisite 1
+- [ ] Prerequisite 2
+- [ ] Component 1
+- [ ] Component 2
+- [ ] Docs
+- [ ] Tests
+
+
+
+## Summary/Abstract
+
+Provide a high-level summary of the proposed change. Keep it short.
+
+This design process is usually around adding a series of standards around recording change notes for the project. The intent is to make the process of building the release changelog faster and easier and track decisions for the future. It is usually a combination of a feature and effort-tracking document, a product requirements document, and a design document.
+
+## Background/Motivation
+
+Provide the motivation for the feature and any context required to understand the motivation. The motivation should justify any potential impact and why it is important. Often these proposals start as an issue, forum post, email and it's helpful to link to it in this section to provide context and credit the right people for the idea.
+
+It is vital for projects to be able to track the chain of custody for a proposed enhancement from conception through implementation which can sometimes be difficult to do in a single Github issue, especially when it is a larger design decision or cuts across multiple areas of the project.
+
+The purpose of the design proposal processes is to reduce the amount of "siloed knowledge" in a community. By moving decisions from a smattering of mailing lists, video calls, slack messages, Github exchanges, and hallway conversations into a well tracked artifact, the process aims to enhance communication and discoverability.
+
+## (Optional) User/User Story
+
+Detail the things that people will be able to do with the feature if it is implemented. Include as much detail as possible so that people can understand the "how" of the system. This can also be combined with the background/motivation above.
```

Comment:
```suggestion
Detail the things that people will be able to do with the feature if it is implemented. Include as much detail as possible so that people can understand the "how" of the system. This can also be combined with the prior sections.
```

Reply:
I'd also add something about who the target users that will be using this feature are.